crates/water/src/runtime/core.rs

```diff
@@ -37,10 +38,13 @@ impl H2O<Host> {
         }
 
         let engine = Engine::new(&wasm_config)?;
-        let linker: Linker<Host> = Linker::new(&engine);
-
+        
         let module = Module::from_file(&engine, &conf.filepath)?;
+        
+        let linker: Linker<Host> = Linker::new(&engine);
 
+        // linker.allow_unknown_exports(true);
+        
```

Comment:
Are these just pure formatting or is the ordering making any differences?

Reply:
If the ordering matters, it is better to leave some explicit comments explaining so. If just styling/formatting, I guess it is fine to leave it like that.

Reply:
The order doesn't matter, it's just styling :D.